homeassistant/components/squeezebox/media_player.py

```diff
@@ -234,11 +211,25 @@ def __init__(self, player: Player, server: Server) -> None:
             identifiers={(DOMAIN, self._attr_unique_id)},
             name=player.name,
             connections={(CONNECTION_NETWORK_MAC, self._attr_unique_id)},
-            via_device=(DOMAIN, server.uuid),
+            via_device=(DOMAIN, coordinator.server_uuid),
             model=player.model,
             manufacturer=_manufacturer,
         )
 
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        if self._previous_media_position != self.media_position:
+            self._previous_media_position = self.media_position
+            self._last_update = utcnow()
+        self._attr_available = self.coordinator.available
+        self.async_write_ha_state()
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self._attr_available and super().available
```

Comment:
What is the content of `self._attr_available`? If it is always `self.coordinator.available` why don't we use that directly

Reply:
You're right, this is a legacy from before we had a coordinator. We can use the coordinator attribute directly.